scripts/parseCodePath.ts

```diff
@@ -26,6 +26,8 @@ interface elementChoice {
 export async function parse(xml: string) {
   const { modelInfo } = await xml2js.parseStringPromise(xml);
   const domainInfo = modelInfo.typeInfo.filter((ti: any) => ti.$.baseType === 'FHIR.DomainResource');
+  const elementInfo = modelInfo.typeInfo.filter((ti: any) => ti.$.baseType == 'FHIR.Element' && !ti.$.identifier);
```

Comment:
```suggestion
  const elementInfo = modelInfo.typeInfo.filter((ti: any) => ti.$.baseType === 'FHIR.Element' && !ti.$.identifier);
```
I'm guessing we want === here unless this was intentional?

Reply:
Yes you are right!